### PR TITLE
v7.6 packaging changes

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for:
- https://github.com/mattermost/mattermost-server/pull/21443
- https://github.com/mattermost/mattermost-webapp/pull/11346

Summary of changes:
- Updated [GitLab SSO](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6104/onboard/sso-gitlab.html), [Google SSO](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6104/onboard/sso-google.html), and [OpenID Connect SSO](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6104/onboard/sso-openidconnect.html) pages to add free on cloud badge
- Updated [Insights](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6104/welcome/insights.html) page to mark My Insights as free on cloud, and [Team Insights as paid](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6104/welcome/insights.html#team-insights).
- Updated Cloud Limitations page to [clarify group calls free on cloud](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6104/onboard/mattermost-limits.html#mattermost-cloud-free-limits), remove file storage limits, & remove Professional section altogether
- Updated [Shared Channels badge](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6104/onboard/shared-channels.html) from Enterprise to Professional & Enterprise
- Updated [Editions & Offerings](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/6104/about/editions-and-offerings.html) page based on above changes

### CODE REVIEW

File: source/onboard/mattermost-limits.rst [hunk 1]
```diff
@@ -14,19 +14,21 @@ In addition to these limits, visit our pricing page to see a complete list of fe
...
    These limits do not apply to self-hosted deployments. 
 
-Mattermost Free limits
-----------------------
+Mattermost Cloud Free limits
+----------------------------
 
 **Platform limits**
 
-- Maximum 1 team. Any additional teams created during a trial or on a paid plan will be archived.
```
Comment (cwarnermm) [inline source/onboard/mattermost-limits.rst:22]: @johndavidlugtu - Can you please confirm that this line should be removed from the docs please?

File: source/onboard/mattermost-limits.rst [hunk 2]
```diff
@@ -14,19 +14,21 @@ In addition to these limits, visit our pricing page to see a complete list of fe
...
-- 10 GB file storage across the platform, with 100 MB upload limit. The oldest files that exceed this limit will be archived.
```
Comment (cwarnermm) [inline source/onboard/mattermost-limits.rst:23]: @johndavidlugtu - Can you please confirm that this line should be removed from the docs please?

File: source/onboard/mattermost-limits.rst [hunk 3]
```diff
@@ -36,13 +38,6 @@ Mattermost Free limits
    
    Mattermost reserves the right to archive or delete messages and establish or change (1) limits as to how many messages can be stored, (2) how long such messages will be stored, and (3) charges for storing such messages. Mattermost further reserves the right to establish or change limits on periods of inactivity that may result in the termination of your messages cloud storage and deletion or archiving of any stored messages.
 
-Mattermost Professional limits
```
Comment (cwarnermm) [inline source/onboard/mattermost-limits.rst:39]: @johndavidlugtu - Can you please confirm that this line should be removed from the docs please?

File: source/onboard/shared-channels.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 Shared channels (experimental)
 ==============================
 
-.. include:: ../_static/badges/ent-cloud-selfhosted.rst
+.. include:: ../_static/badges/pro-cloud-selfhosted.rst
```
Comment (cwarnermm) [inline source/onboard/shared-channels.rst:4]:
```suggestion
.. include:: ../_static/badges/ent-pro-cloud-selfhosted.rst
```

File: source/onboard/mattermost-limits.rst (hunk 1 above)
Comment (johndavidlugtu) [reply source/onboard/mattermost-limits.rst:22]: @cwarnermm 10GB file storage limit will now be 1GB as per this [post](https://community.mattermost.com/private-core/pl/o4nohhu1zjbtmcpqo9ekx8x8nr)

Will get back to you re: team limits.

File: source/onboard/mattermost-limits.rst (hunk 3 above)
Comment (johndavidlugtu) [reply source/onboard/mattermost-limits.rst:39]: I believe we would no longer have Professional file limits, or at least we won't be enforcing the 250GB file storage limit. @esethna to verify

**Review by laneycs — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

File: source/onboard/mattermost-limits.rst (hunk 1 above)
Comment (johndavidlugtu) [reply source/onboard/mattermost-limits.rst:22]: Hey @cwarnermm !

Team limit is kept at 1 team

cc: @laneycs

File: source/onboard/mattermost-limits.rst (hunk 2 above)
Comment (johndavidlugtu) [reply source/onboard/mattermost-limits.rst:23]: @cwarnermm the limit is now 1GB for Cloud as per this [GSheet](https://docs.google.com/spreadsheets/d/1mjXnrPuunAiE9QIcVldOvX2s6_Ih49uUs5OHoFwxzc8/edit#gid=1538757693)

**Comment by johndavidlugtu** [comment]
> Up to 8 participants per channel can join calls that last up to 40 minutes.

Screen Shot 2022-12-16 at 9 55 38 AM

@itao  Will the UX be that the icon notifying users it's a paid feature be only visible in a channel with more than 8 participants? Or will the icon be phased out in exchange for some form of notification once inside a call?

File: source/onboard/mattermost-limits.rst (hunk 1 above)
Comment (cwarnermm) [reply source/onboard/mattermost-limits.rst:22]: Thanks, @johndavidlugtu! Brought these bullets back and updated them where needed.

File: source/onboard/mattermost-limits.rst [hunk 5]
```diff
@@ -14,19 +14,23 @@ In addition to these limits, visit our pricing page to see a complete list of fe
...
 
 **Channels limits**
 
 - 10,000 message history limit. The oldest messages that exceed this limit will be archived.
 
+**Calls limits**
+
+- Up to 8 participants per channel can join calls that last up to 40 minutes.
+
 **Boards limits**
```
Comment (wuwinson) [inline source/onboard/mattermost-limits.rst:34]: @cwarnermm All Boards limits will be removed for v7.7, so there will no longer be any limits of:
- Maximum 5 saved views per board.
- Only the 500 most recently updated cards on the server are displayed.

This entire section can be removed for the next release. Thanks!

Comment (cwarnermm) [reply source/onboard/mattermost-limits.rst]: Thanks, @wuwinson! I've removed the Boards limits from the docs page in a new commit.

**Review by wuwinson — APPROVED** [review]

**Comment by justinegeffen** [comment]
Also includes changes for https://github.com/mattermost/mattermost-server/pull/21882

**Comment by justinegeffen** [comment]
Also addresses https://github.com/mattermost/mattermost-server/pull/21556

File: source/onboard/mattermost-limits.rst (hunk 3 above)
Comment (johndavidlugtu) [reply source/onboard/mattermost-limits.rst:39]: @cwarnermm based on [checklist](https://docs.google.com/spreadsheets/d/1mjXnrPuunAiE9QIcVldOvX2s6_Ih49uUs5OHoFwxzc8/edit#gid=1538757693&range=B19) we'll continue to have a Professional file limit, but not enforce it. Note that this is for Cloud only.

Deferring to you how/if we want to surface this

Comment (johndavidlugtu) [reply source/onboard/mattermost-limits.rst:39]: Actually @cwarnermm it looks like the limit is gone for Cloud Pro altogether.

Reference: https://docs.google.com/presentation/d/11hYF5p5sZ_mBi3lF1eT3tHJSM6Iw4uOFkhQG-VK5uPc/edit?disco=AAAAnYSbYJA

Comment (johndavidlugtu) [reply source/onboard/mattermost-limits.rst:39]: Apologies for the back and forth!

**Comment by itao** [comment]
> Will the UX be that the icon notifying users it's a paid feature be only visible in a channel with more than 8 participants

This screenshot was from previous packaging where all channel calls, except DMs, required a paid plan. In the new packaging, the call button will not show the key icon for upgrade and instead only become disabled if 8 participants are already on the call.

**Pull request merged**